### PR TITLE
added new html files to be the messages or widows

### PR DESCRIPTION
**Pull request by rabeedameer**
please review!!

### CODE REVIEW

File: single-right.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <link rel="stylesheet" href="/styles/main.css">
+  <title>Guessing Game</title>
+</head>
+<body>
+  <main class="main-content jumbotron jumbotron-fluid mb-0 d-flex">
+    <div class="container text-center m-auto">
+      <h1 class="display-1">Great job!</h1>
+      <p class="lead" id="success"></p>
+
+      <button class="btn btn-primary btn-lg" role="button" id="play" onclick="writeResult(guessingGame())"><a href="index.html"></a>Play again!</button>
```
Comment (Majed187) [inline single-right.html]: change the id name :(

**Review by Majed187 — CHANGES_REQUESTED** [review]
change the name of id

**Review by rabeedameer — COMMENTED** [review]
i changed the names of IDs and added new ones

**Review by Majed187 — COMMENTED** [review]
:+1:

**Pull request merged**